.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,2 @@
+blank_issues_enabled: false
+
```

Comment:
Thanks for adding this stub.

There is a blank line at the end of the file that is causing a sanity error. Please delete the blank line to eliminate the sanity error. That will clear the way for approving and merging this PR.

Reply:
@kerry-meyer Have fixed the issue on sanity error. And the checks have passed

Reply:
Thanks for this change and all of your work on the templates.

I will mark this issue "Resolved" and will approve and merge the content for this PR.